dashboard_viewer/docker-init.sh

```diff
@@ -7,9 +7,11 @@ wait-for-it "$POSTGRES_ACHILLES_HOST:$POSTGRES_ACHILLES_PORT"
 
 # Apply django migrations
 python manage.py migrate
-python manage.py migrate --database=achilles uploader
-python manage.py migrate --database=achilles materialized_queries_manager
-python manage.py populate_countries
+python manage.py migrate --database=achilles
```

Comment:
Check if this is enough to apply all migrations of all apps associated with the ACHILLES datasource.
Maybe some changes have to be made to the custom router.